### DISCUSSION TITLE
What can I do after forgot login password ?

### DISCUSSION THREAD

**Discussion by hailinlu** [electerm/electerm#3515] [Q&A] [answered]
I have forgot the password ,but I think I input the true pwd,how can I find my password 
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/69ccd16c-b2bc-45ce-97af-19e8b3034576">

**Comment by zxdong262** [comment, marked as answer]
```rm ~/Library/Application\ Support/electerm/users/default_user/electerm.data.nedb```

**Reply by hailinlu** [reply]
thank you for helping, it's working. @zxdong262
